README.md

```diff
@@ -1,31 +1,29 @@
 # Visit someone in prison
 
-This is a full rewrite from the ground up, using a database instead of
-serialised data in URLs in emails. The CSS and JavaScript has largely been
-ported intact from the previous application.
+This application contains the public interface for booking a prison visit.
 
-The source of the predecessor can be found at
-[ministryofjustice/prison-visits](https://github.com/ministryofjustice/prison-visits).
+It is stateless and relies entirely on the prison visits booking API exposed by [ministryofjustice/prison-visits-2](https://github.com/ministryofjustice/prison-visits-2), which exposes prison information, slot availability, and allows booking and managing a visit.
 
-## Notes on models
+The codebase was split from [ministryofjustice/prison-visits-2](https://github.com/ministryofjustice/prison-visits-2), which previously was also responsible for serving the public interface.
 
-### Slots
+## The API into prison-visits-2
 
-#### `RecurringSlot`
+Details of the API methods consumed can be found in [api.rb](app/services/prison_visits/api.rb).
 
-Represents a time period with begin and end hours and minutes. It does not have
-a day or date.
+### Local development
 
-#### `ConcreteSlot`
+The default configuration of `PRISON_VISITS_API` is http://localhost:3000/, so the least-friction development approach is to run this app on another port.
 
-The expression of a slot on a particular day: this is a unique time that can be
-booked.
+          pvb2 $ rails server
+    pvb-public $ rails server -p 4000
```

Comment:
I will do, when I've written the readme for PVB2. I was going to wait until we remove all the public stuff from it if that's ok.